ponder.schema.ts

```diff
@@ -40,3 +41,24 @@ export const swap = onchainTable("swap", (t) => ({
 export const swapRelations = relations(swap, ({ one }) => ({
   pool: one(pool, { fields: [swap.poolId], references: [pool.poolId] }),
 }));
+
+export const position = onchainTable("position", (t) => ({
+  positionId: t.hex().primaryKey(),
+  poolId: t.hex().notNull(),
+  owner: t.hex().notNull(),
+  tickLower: t.bigint().notNull(),
+  tickUpper: t.bigint().notNull(),
+  liquidity: t.bigint().notNull(),
+  salt: t.hex().notNull(),
+  chainId: t.integer().notNull(),
+}),
+  (table) => ({
+    poolIdIndex: index().on(table.poolId),
+    ownerIndex: index().on(table.owner),
+    chainIdIndex: index().on(table.chainId),
```

Comment:
can add this to the other tables as well

Reply:
ah yea thatd be good; I was planning on looking into compound indexes too i.e. poolId-chainId is maybe the common call pattern